springBreakForm.php

```diff
@@ -0,0 +1,252 @@
+<?php
+
+session_cache_expire(30);
+session_start();
+ini_set("display_errors", 1);
+error_reporting(E_ALL);
+
+$loggedIn = false;
+$accessLevel = 0;
+$userID = null;
+
+if(isset($_SESSION['_id'])){
+    $loggedIn = true;
+    $accessLevel = $_SESSION['access_level'];
+    $userID = $_SESSION['_id'];
+}
+
+
+// include the header .php file s
+if($_SERVER['REQUEST_METHOD'] == "POST"){
+    require_once('include/input-validation.php');
+    //require_once('database/dbSpringBreakForm.php');
+    $args = sanitize($_POST, null);
+    $required = array("email", "name", "school_date", "isAttending", "hasWaiver", "questions_comments");
+    if(!wereRequiredFieldsSubmitted($args, $required)){
+        echo "Not all fields complete";
+        die();
+    }else {
+        foreach($args as $key => $val){
+            echo "{$key}:" . " " . "{$val}" . "<br>";
+        }
+    }
+}
+
+
+?>
+
+<html>
+
+<head>
+    <?php include_once("universal.inc")?>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Stafford Junction | Spring Break Form</title>
+</head>
+<style>
```

Comment:
If you have the time, I would probably put the css in the base.css file just to keep it a little bit cleaner